deployment/plur_push_service/templates/service.yaml

```
@@ -0,0 +1,16 @@
+  namespace: plur-push
+  labels:
+    app.kubernetes.io/part-of: plur-push
+    app.kubernetes.io/managed-by: Helm
+spec:
+  selector:
+    app.kubernetes.io/name: plur-push
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8000
+  type: ClusterIP
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)